scripts/gen_vimdoc.py

```diff
@@ -937,14 +928,9 @@ def fmt_doxygen_xml_as_vimhelp(filename, target):
 
         func_doc = "\n".join(split_lines)
 
-        if 'Deprecated' in xrefs:
-            deprecated_fns_txt[name] = func_doc
```

Comment:
?

Reply:
It's done in once in line 855 for all xrefs instead of splitting it up.